src/natcap/invest/wind_energy.py

```diff
@@ -2405,6 +2405,11 @@ def _clip_vector_by_vector(
         base_layer_defn.GetName(), base_layer.GetSpatialRef(), base_geom_type)
     base_layer.Clip(clip_layer, target_layer)
 
+    empty_clip = False
+    if target_layer.GetFeatureCount() <= 0:
```

Comment:
What would cause `GetFeatureCount()` to return something `< 0`?

Reply:
Oh, [I see](https://gdal.org/python/osgeo.ogr.Layer-class.html#GetFeatureCount) what could cause this.  Could you add a comment about the `< 0` case here for future us?

Reply:
Added a comment here: https://github.com/natcap/invest/pull/475/commits/a105ac775642caaf175093dc130a9257b86e2901. Honestly, I think I meant to write `== 0`, and just got lucky about it being possible to be `-1` 😄